cmd/open.go

```diff
@@ -0,0 +1,56 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/CircleCI-Public/circleci-cli/git"
+	"github.com/jtacoma/uritemplates"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+)
+
+func projectUrl(remote *git.Remote) (string, error) {
+
+	template, err := uritemplates.Parse("https://app.circleci.com/{vcs}/{org}/{project}/pipelines")
```

Comment:
This lib looks a bit old, I dunno how else to tell if it's a decent choice.

`url.PathEscape` might be a safer bet

Reply:
I'd equate old with "stable" in this case – it's a small library that does a straightforward task.

Reply:
Is the new dependancy, values map and extra error handling really better than..

```
func projectUrl(remote *git.Remote) string {
	return fmt.Sprintf("https://app.circleci.com/%s/%s/%s/pipelines",
		url.PathEscape(strings.ToLower(string(remote.VcsType))),
		url.PathEscape(remote.Organization),
		url.PathEscape(remote.Project))
}
```